accounting/fiscality/taxes/fiscal_positions.rst

```diff
@@ -0,0 +1,110 @@
+==========================================
+Fiscal Positions (tax and account mapping)
+==========================================
+
+Default taxes and accounts are set on products and customers to create new transactions on the fly.
+However, you might have to use different taxes and record the transactions on different accounts,
+according to your customers' and providers' localizations and business types.
+
+**Fiscal Positions** allow you to create *sets of rules* to automatically adapt the taxes and the
+accounts used for a transaction.
+
+.. image:: media/fiscal-positions-intra-community.png
+   :align: center
+   :alt: Example: Belgian to Intra-Community tax mapping with Fiscal Positions in Odoo Accounting
+
+They can be applied in various ways:
+
+- :ref:`automatically applied, based on some rules <fiscal_positions/automatic>`
+- :ref:`manually applied on a transaction <fiscal_positions/partner>`
+- :ref:`assigned to a partner, on its contact form <fiscal_positions/transaction>`
+
+.. note::
+   A few Fiscal Positions are already preconfigured on your database, as part of your :doc:`Fiscal
+   Localization Package <../../fiscal_localizations/overview/fiscal_localization_packages>`.
+
+Configuration
+=============
+
+Tax and Account Mapping
+-----------------------
+
+To edit or create a Fiscal Position, go to :menuselection:`Accounting --> Configuration --> Fiscal
+Positions`, and open the entry you want to modify or click on *Create*.
+
+The mapping of taxes and accounts is based on the default taxes and accounts defined in the
+products' forms.
+
+- To map to another tax or account, fill out the right column.
+- To remove a tax, rather than replacing it with another, leave the field **Tax to Apply** empty.
+- To replace a tax with multiple other taxes, add multiple lines with the same **Tax on Product**.
+
+.. note::
+   The mapping only works with *active* taxes. Make sure that all the taxes you need are active by
+   checking them in :menuselection:`Accounting --> Configuration --> Taxes`.
+
+.. _fiscal_positions/automatic:
+
+Automatic application
+---------------------
+
+You can configure your Fiscal Positions to be applied automatically, following a set of conditions.
+
+To do so, open the Fiscal Position you want to modify and click on **Detect Automatically**. A set
+of conditions can be configured:
+
+- **VAT Required**: The VAT number *must* be indicated in the customer's contact form.
+- **Country Group** / **Country**: The Fiscal Position is applied to these countries.
+
+.. image:: media/fiscal-positions-automatic.png
+   :align: center
+   :alt: Example of settings to apply a Fiscal Position automatically
+
+.. note::
+   Taxes on **eCommerce orders** are automatically updated once the visitor has logged in or filled
+   out their billing details.
```

Comment:
```suggestion
   Taxes on **eCommerce orders** are automatically updated once the visitor has logged in or filled
   out his billing details.
```

Reply:
I use "their" in a gender-neutral way. The customer here is gender-neutral, and not a man.
Used this way, "their" is singular and not plural. 👍